SeafFileProvider/FileProvider.m

```diff
@@ -183,14 +183,23 @@ - (void)itemChangedAtURL:(NSURL *)url
     NSFileProviderItemIdentifier identifier = [self persistentIdentifierForItemAtURL:url];
     Debug("File changed: %@ %@", url, identifier);
     SeafItem *item = [[SeafItem alloc] initWithItemIdentity:identifier];
+    if ([self readFromLocal:identifier]) {
```

Comment:
`SeafItem *item = [self readFromLocal:identifier];
if (!item) {
    item = [[SeafItem alloc] initWithItemIdentity:identifier];
}`
It is better in this way.